arm/data/track

```diff
@@ -0,0 +1,4 @@
+This is here so that this directory, arm/data, gets added to the git repo,
+since git will not track empty directories. This is where any data generated
+by pianotuner as it runs gets saved. Right now, it's just debug data, like
+autocorrelation snapshots.
```

Comment:
OK. Why worry about having an empty dir in the repo at all? If the dir is missing, the application should create it on start.

Reply:
It was easier to make this file than to figure out how test for missing directories then make directories using the C standard library. Like a /lot/ easier. And this is only for debugging anyway.